bento2seldon4recsys/monitoring.py

```diff
@@ -8,39 +10,49 @@ class RecommenderMonitor(Monitor):
     def __init__(self, bento_service: BentoService) -> None:
         super().__init__(bento_service)
 
-        self._ndcg = self._create_metric(
```

Comment:
Same comment here as the one in the other PR. Instead of lazily initializing the metrics, receive the list of extra labels in the constructor.